README.md

```diff
@@ -1,24 +1,135 @@
+    </li>
+    <li><a href="#endpoints">Endpoints</a></li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+  </ol>
+</details>
 
-Things you may want to cover:
+### About The Project
+## Background and Description
+Mesh is the capstone project for the Turing School of Software and Design. A platform to standardize and organize feedback for programming schools and bootcamps alike. Students are able to easily retrieve and reflect upon past feedback to better improve for the future! 
+## Learning Goals
+- Work with a Full Stack team to integrate and learn each others workflow and framework
+- Learn and work with new frameworks or languages
+- Use a professional workflow to reflect industry standards
+### Built With
 
-* Ruby version
+* [Ruby on Rails](https://rubyonrails.org)
 
-* System dependencies
 
-* Configuration
+## Database Design Document
+![Database Design Document](https://imgur.com/QTy9zsE.png)
 
-* Database creation
 
-* Database initialization
+<!-- GETTING STARTED -->
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+This is an example of how to list things you need to use the software and how to install them.
+* ruby
+  ```sh
+  rbenv install 2.5.3
+  ```
+* rails
+  ```sh
+  gem install rails --version 5.2.5
+  ```
+* postgreSQL
+  ```sh
+  install link: https://www.postgresql.org/download/
+  ```
+
+### Installation
+
+1. Clone the repo
+   ```sh
+   git clone git@github.com:Turing-Mesh/mesh-backend.git
+   ```
+2. Bundle Install
+   ```sh
+   bundle install
+   ```
+3. (Optional Heroku deployment)
+   ```sh
+   heroku install guide: https://devcenter.heroku.com/articles/git
+   ```
+
+<!-- ENDPOINTS -->
+### Endpoints
+> For more details please see our [API Contract](API_contract.md)
+
+| HTTP verbs | Paths  | Used for | Output |
+| ---------- | ------ | -------- | ------:|
+| GET | /api/v1/students/:student_id/student_projects?mod=1 | Get all of the project feedback for single student and a single mod | [json](#student-projects-by-mod) |
+| PATCH | /api/v1/students/:student_id/student_projects/:id | Update student's project with personal comments | [json](#students-update-project) |
+| GET | /api/v1/instructors/:instructor_id/instructor_students?mod=1 | Get all of the student names and ids for instructor's current mod | [json](#instructor-module-students) |
+| POST | /api/v1/instructors/:instructor_id/instructor_students/search | Search for a single student's project information, response is same as students-project-by-mod (we need first and last name to search by in post body request)| [json](#instructor-students-search) |
+| POST | /api/v1/users | Registration new user  | [json](#user-registration) |
+| POST | /api/v1/sessions | Login a user | [json](#sessions-create) |
+| ERROR | errors | Error handling for requests | [json](#error-handling) |
+
+<!-- USAGE EXAMPLES -->
```

Comment:
Do we maybe just want to link to the API Contract here to avoid maintaining usage examples in two places?